website/src/pages/downloads/macos.tsx

```diff
@@ -1,112 +1,21 @@
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import { usePluginData } from '@docusaurus/useGlobalData';
```

Comment:
Nice method; didn't know it exists, from the top of memory the Storybook plugin generate a JSON file, and the related components imports it, but this is nice